### PR TITLE
Port integrations over from old site

### PR DESCRIPTION
**Pull request by mghill**
This page has been maintained and is relatively current. Very minor edits, plus a hefty drop into textlintrc. Needs a merge and then should be ready to go straight off

### CODE REVIEW

File: reference/supported-integrations.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Messaging
+* Email (MS Graph, Gmail, SMTP, AWS SES)
+* SMS (MessageMedia)
+* MQTT
+* Webhooks
+* Node-RED
+
+## Resource Booking
+* Exchange Online (via MS Graph API)
```
Comment (kimburgess) [inline reference/supported-integrations.md]:
```suggestion
* Microsoft Office365
```

File: reference/supported-integrations.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+* GSuite (Google Calendar API)
```
Comment (kimburgess) [inline reference/supported-integrations.md]:
```suggestion
Google Workspace
```

File: reference/supported-integrations.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+* IBM Domino
```
Comment (kimburgess) [inline reference/supported-integrations.md]:
```suggestion
```
We do not have a current integration for this.

**Comment by mghill** [comment]
alright this is pretty much ready to merge. I gave it a quick once over, @jeremyw24 want a quick look first?

**Comment by jeremyw24** [comment]
LGTM

**Pull request merged**